src/ai/backend/manager/api/session.py

```diff
@@ -132,6 +132,7 @@
 log = BraceStyleAdapter(logging.getLogger(__name__))
 
 _json_loads = functools.partial(json.loads, parse_float=Decimal)
+session_ptask_group: aiotools.PersistentTaskGroup | None = None
```

Comment:
This should be inside the per-app context.

Reply:
Could you mangle its name with more semantics and split following the purposes?
e.g., `dbtxn_ptaskgroup`, `rpc_ptaskgroup`, ...